backend/services/controlnet_service.py

```
@@ -281,20 +294,20 @@ async def _transform_with_huggingface(
             if response.status_code == 200:
                 # HF returns image bytes directly
                 result_bytes = response.content
-                logger.info("✅ Hugging Face ControlNet transformation completed")
+                logger.info("✅ Hugging Face img2img transformation completed")
                 return result_bytes
             elif response.status_code == 404:
-                error_msg = f"ControlNet model not found: {model_name}. Check if model exists on HuggingFace."
+                error_msg = f"HF img2img model not found: {model_id}. Check if model exists on HuggingFace."
                 logger.error(f"❌ {error_msg}")
-                logger.error(f"💡 Available models should be: control_v11p_sd15_openpose, control_v11f1p_sd15_depth, etc.")
+                logger.error(f"💡 Available models: runwayml/stable-diffusion-v1-5, stabilityai/stable-diffusion-2-1, etc.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)